components/EventDate/src/stories/eventDate.bem.stories.mdx

```diff
@@ -0,0 +1,49 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+import pkg from "../../package.json";
+import "../index.css";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS Components/Data Display/EventDate"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="EventDate" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <p class="denhaag-event-date">
+      <svg
+        class="denhaag-event-date__icon"
+        width="16"
+        height="17"
+        viewBox="0 0 16 17"
+        fill="none"
+        xmlns="http://www.w3.org/2000/svg"
+      >
+        <path
+          d="M5.5 0.667969C5.96024 0.667969 6.33333 1.04106 6.33333 1.5013V2.33464H9.66667V1.5013C9.66667 1.04106 10.0398 0.667969 10.5 0.667969C10.9602 0.667969 11.3333 1.04106 11.3333 1.5013V2.33464H13.8333C14.7538 2.33464 15.5 3.08083 15.5 4.0013V14.8346C15.5 15.7551 14.7538 16.5013 13.8333 16.5013H2.16667C1.24619 16.5013 0.5 15.7551 0.5 14.8346V4.0013C0.5 3.08083 1.24619 2.33464 2.16667 2.33464H4.66667V1.5013C4.66667 1.04106 5.03976 0.667969 5.5 0.667969ZM4.66667 4.0013H2.16667V6.5013H13.8333V4.0013H11.3333V4.83464C11.3333 5.29487 10.9602 5.66797 10.5 5.66797C10.0398 5.66797 9.66667 5.29487 9.66667 4.83464V4.0013H6.33333V4.83464C6.33333 5.29487 5.96024 5.66797 5.5 5.66797C5.03976 5.66797 4.66667 5.29487 4.66667 4.83464V4.0013ZM13.8333 8.16797H2.16667V14.8346H13.8333V8.16797Z"
+          fill="#1261A3"
+        />
+      </svg>
+      <span class="denhaag-posttype-label denhaag-event-date__text">vrijdag 14 oktober</span>
```

Comment:
This should be the <time> element:  https://www.w3schools.com/tags/tag_time.asp since we're talking about a publish/modified date.